src/libpng-1.6.35.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="libpng-1.6.35" name="PNG Reference
+   Library License v1 (for libpng 0.5 through 1.6.35" listVersionAdded="">

```

Comment:
Please remove the line break from "name" attribute.
And add value to "listVersionAdded".

Reply:
thanks!